### PR TITLE
Replaced check_validity(..,false) with check_invalidity(..)

### PR DESCRIPTION
**Pull request by prachi1210**
Closes https://github.com/coala/coala-bears/issues/1729

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by jayvdb** [comment]
Hi @prachi1210 , you do not need to close this Pull Request.  The messages above by gitmate list things to improve, and then you resubmit your changes to *this* Pull Request.  You can find more info about this at http://coala.io/newcomer#step-8-review-process , or ask for guidance on http://coala.io/chat .
If you just close this PR, and start a new one, you are missing many important skills that you need to become a coala developer.  You dont want to miss learning those skills, and we dont want that either.  We're here to help.

**Comment by prachi1210** [comment]
@jayvdb okay, will have a look

File: tests/general/SpaceConsistencyBearTest.py [hunk 1]
```diff
@@ -26,26 +26,26 @@ def test_defaults(self):
...
 
         self.check_validity(self.uut, ['    t'])
-        self.check_validity(self.uut, ['\tt'], valid=False)
-        self.check_validity(self.uut, ['t \n'], valid=False)
-        self.check_validity(self.uut, ['t'],
-                            force_linebreaks=False, valid=False)
+        self.check_invalidity(self.uut, ['\tt'])
+        self.check_invalidity(self.uut, ['t \n'])
+        self.check_invalidity(self.uut, ['t'],
+                            force_linebreaks=False)
```
Comment (IpshitaC) [inline tests/general/SpaceConsistencyBearTest.py]: @prachi1210 This is just an issue of improper indentation. Add proper indentation to line 32, currently it's under indented. You could apply the following patch to fix it directly.

File: tests/general/SpaceConsistencyBearTest.py [hunk 2]
```diff
@@ -62,13 +62,11 @@ def test_enforce_newline_at_eof(self):
                              "    print('funny')\n",
                              "    print('funny end.')\n"],
                             force_linebreaks=False)
-        self.check_validity(self.uut,
+        self.check_invalidity(self.uut,
                             [' no hello world'],
```
Comment (IpshitaC) [inline tests/general/SpaceConsistencyBearTest.py]: @prachi1210 The same issue here too. Simply apply this patch to fix it directly.

**Comment by Monal5031** [comment]
@prachi1210 Try running coala locally that might help in resolving the issue 😃

**Comment by yash-nisar** [comment]
That is what we've done here @IpshitaC . IMO, it is perfectly fine and we shouldn't be too picky about the commit message, afterall this is a newcomer issue and the commit description states the purpose of the commit. :+1:

**Comment by IpshitaC** [comment]
@yash-nisar according to the [Writing Good Commits guide](http://api.coala.io/en/latest/Developers/Writing_Good_Commits.html), the commit description should be written in present imperative tense and I think it should be adhered to. Just a small update required. Otherwise, this issue is perfectly fine. 😄

**Comment by yash-nisar** [comment]
@IpshitaC We've followed that convention here. From the docs : 
> Written in imperative present tense (i.e. Add something, not Adding something or Added something).

The sentence below is already written in imperative mood.
```
Replace all instances of check_validity(..,valid=false)
with check_invalidity(..)
```

**Comment by prachi1210** [comment]
@Udayan12167 your views on this?

**Comment by yash-nisar** [comment]
Rebase required.

**Comment by RJ722** [comment]
ack 6ba79eba8c8388847d5452c6eb24fd4f373065d5

**Comment by RJ722** [comment]
ack 6ba79eba8c8388847d5452c6eb24fd4f373065d5

**Comment by prachi1210** [comment]
@RJ722 done

**Comment by yash-nisar** [comment]
You haven't rebased yet, your branch is a few commits behind master. CC @prachi1210

**Comment by prachi1210** [comment]
@yash-nisar CI builds are failing after rebase.

**Comment by yash-nisar** [comment]
@prachi1210 Don't worry, the build failures are unrelated to your issue. 
* Regarding the Travis failure, we have opened an [issue](https://github.com/coala/coala-bears/issues/1864) which is likely to get resolved quickly. :smile: 
* The circle and Appveyor build failure is due to the timeout factor while installing dependencies and most probably should vanish if you trigger a rebuild. (happens often)

**Comment by yash-nisar** [comment]
The problem has been [fixed](https://github.com/coala/coala-bears/commit/e44890958290deb79a90d1440543825a962aa3f9). Rebase again and your tests shall pass :+1:

**Comment by yash-nisar** [comment]
@prachi1210 Feel free to ping me on gitter for any help, we need to merge this asap. So, let us know when you're done with the rebase. :smile:

**Comment by prachi1210** [comment]
@yash-nisar Travis build is still failing

**Comment by prachi1210** [comment]
@yash-nisar done from my end.

**Comment by yash-nisar** [comment]
ack 1067ed77e13dc0be658cb8190cabb0c59e89547c

**Comment by jayvdb** [comment]
unack 1067ed7

**Comment by jayvdb** [comment]
ack a9bd165

**Comment by jayvdb** [comment]
needs a rebase.

**Comment by prachi1210** [comment]
@jayvdb the builds are failing after a rebase.

**Comment by prachi1210** [comment]
@yash-nisar any ideas why the CircleCI build is failing?

**Comment by yash-nisar** [comment]
The failure is related to the DocumentationStyleBear which should have been fixed. CC @damngamerz who is working on this as his GSoC project. 

One more rebase (sorry :stuck_out_tongue: ) and the errors should be gone :smile:

**Comment by yash-nisar** [comment]
Btw you have 2 commits for a single change. The commits are atomic and one commit should describe a single change. So squash your commits and amend your message as @jayvdb suggested. 

Also ping us on [gitter](https://coala.io/chat) if you face any difficulties. :)

**Comment by damngamerz** [comment]
@yash-nisar It shows
`AttributeError: 'DocumentationComment' object has no attribute 'position'`
IMO there should be no reason for this to happen. As I made sure to get my upstream patch in `coala` repo to be merged first before this change. Still why only Circle CI is failing :/

**Comment by yash-nisar** [comment]
@prachi1210 Try an uncached rebuild.

**Comment by prachi1210** [comment]
@yash-nisar  CircleCI is failing again.

**Comment by damngamerz** [comment]
@yash-nisar @jayvdb I see the problem.... either we go for an uncached build or a coala dev version bump to the latest.
```
+ pip install -r requirements.txt
Requirement already satisfied: coala>=0.12.0.dev20170604215434 in /home/ubuntu/.pyenv/versions/3.4.3/lib/python3.4/site-packages (from -r requirements.txt (line 3))
```

**Comment by prachi1210** [comment]
@damngamerz what should I do to resolve this?

**Comment by damngamerz** [comment]
@prachi1210 I have bumped the upstream version... The tree is green again... :tada: 
You should rebase this It will pass now :smile:

**Comment by meetmangukiya** [comment]
ack 39b87aa and @rultor merge

**Comment by meetmangukiya** [comment]
@rultor merge

**Pull request merged**